Source/XCDBLD/MachOType.swift

```diff
@@ -1,7 +1,7 @@
 import Result
 
 /// Describes the type of Mach-O files.
-/// See https://developer.apple.com/library/mac/documentation/DeveloperTools/Reference/XcodeBuildSettingRef/1-Build_Setting_Reference/build_setting_ref.html#//apple_ref/doc/uid/TP40003931-CH3-SW73.
+/// See https://goo.gl/2zdkqZ
```

Comment:
Not sure the obfuscation and indirection justify the link-shortening.

> I don't think making it easy to follow a line length rule should be a goal in itself.
I think this falls under @mdiep’s guidance above.